packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1084,10 +1084,11 @@ export default (Commands, Cypress, cy, state, config) => {
 
           remote = $Location.create(url)
 
-          // if the origin currently matches
+          // if the super origin currently matches
```

Comment:
```suggestion
          // if the super domain origin currently matches
```

Reply:
should be fixed in [2448c3f](https://github.com/cypress-io/cypress/pull/23879/commits/2448c3fab0f7f72ff92f72e954dfba7062bff374)